hydra-node/test/Hydra/HeadLogicSpec.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE PatternSynonyms #-}
 {-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-unused-do-bind #-}
```

Comment:
Shall we remove this pragma?

Reply:
I kept it because this is a test file and it would add some noise to the `step` function